server/handler/base.go

```diff
@@ -88,6 +88,11 @@ func (b *Base) PostStatus(ctx context.Context, prctx pull.Context, client *githu
 	sha := prctx.HeadSHA()
 	base, _ := prctx.Branches()
 
+	if !prctx.IsOpen() {
+		logger.Info().Msg("Skipping status update because PR state is not open")
+		return
+	}
+
```

Comment:
`!IsOpen` seemed better than `IsClosed` since there could be additional options added in the future. We are specifically looking to skip in all cases except when status is `open`, rather than just when it is `closed`.